src/app/components/domain/domain.component.css

```diff
@@ -1,3 +1,6 @@
+:host {
+    flex-grow: 2;
```

Comment:
This would need to be added to every routed component (page-not-found, results, ...).

I suggest that instead you wrap the router-outlet into something (like a `<main>`) and apply style to this element. Something like
```diff
diff --git a/src/app/app.component.html b/src/app/app.component.html
index b696466..5ff291e 100644
--- a/src/app/app.component.html
+++ b/src/app/app.component.html
@@ -1,11 +1,4 @@
 <app-navigation></app-navigation>
 <app-alert></app-alert>
-<router-outlet></router-outlet>
+<main><router-outlet></router-outlet></main>
 <app-footer></app-footer>
-
-
-
-
-
-
-
diff --git a/src/styles.scss b/src/styles.scss
index 23ab227..b7fab34 100644
--- a/src/styles.scss
+++ b/src/styles.scss
@@ -25,3 +25,7 @@ html, body {
   outline: auto;
   box-shadow: none;
 }
+
+main {
+  flex: 2;
+}
```